.github/workflows/release.yaml

```diff
@@ -45,11 +45,41 @@ jobs:
     - name: Upload artifacts
       uses: actions/upload-artifact@v1
       with:
-        name: dist-windows-latest
+        name: x86_64-pc-windows-msvc
         path: ./dist
 
-  dist-ubuntu:
-    name: dist (Ubuntu 16.04)
+  dist-aarch64-pc-windows-msvc:
+    name: dist (aarch64-pc-windows-msvc)
+    runs-on: windows-latest
+    env:
+      RA_TARGET: aarch64-pc-windows-msvc
```

Comment:
wow, one can just cross-compile msvc? nifty!

Reply:
I'm still waiting for a test run, but it works locally if you install the platform-specific build tools.